posthog/views.py

```diff
@@ -183,8 +185,16 @@ def preflight_check(_):
             if not settings.E2E_TESTING
             else False,  # Enables E2E testing of signup flow
             "cloud": settings.MULTI_TENANCY,
+            "ee_available": settings.EE_AVAILABLE,
+            "ee_enabled": is_ee_enabled(),
             "db_backend": settings.PRIMARY_DB.value,
             "available_social_auth_providers": get_available_social_auth_providers(),
             "available_timezones": get_available_timezones_with_offsets(),
+            "opt_out_capture": os.environ.get("OPT_OUT_CAPTURE", False),
+            "posthog_version": VERSION,
```

Comment:
Do we use PostHog version anywhere in logged out capacity? If not, it could be better to not expose it, as this may be a hint regarding security for some malicious actor (potentially).

Reply:
Agreed! I actually think there might even be a bit more extra stuff that could be potentially misused. Now we only returned the most basic attributes when the request is unauthenticated.